### ISSUE TITLE
Wacky brain explosions Vol II.

### ISSUE THREAD

**Issue by ofgulban** [ohbm/hackathon2020#124]
**Title**:
Wacky brain explosions Vol II.
![mind_explosion_inv](https://user-images.githubusercontent.com/4668327/83697079-52f0a400-a5fe-11ea-86ec-4029296012fa.gif)

**Project lead**:
@ofgulban - [Omer Faruk Gulban](http://orcid.org/0000-0001-7761-3727)

**Timezone**:
Late Europe to Americas timezone.

**Hub**:
Europe, Middle East and Africa (but I can only work on this project in the evening/night, therefore I will be more active in the evening/night).

**Description**:
Are you asking to yourself "how can the year 2020 get more **interesting**?". The answer is simple: by using a particle simulator to explode some brains at the comfort of your home, in a completely safe *virtual* environment. 

After concluding my [previous hackaton project](https://github.com/OpenMRBenelux/openmrb2020-hackathon/issues/7) with "more research is needed", here I am planning to continue exploding more brain images in multitude of ways. All in the name of *neuro-science*, of course!

**Link to project**:
[Slowest particle simulator on earth](https://github.com/ofgulban/slowest-particle-simulator-on-earth)

**Mattermost handle**:
`brainexplosions`

**Goals for the OHBM Brainhack**
In the This is a project that was initiated in [OpenMR Benelux 2020 Nijmegen](https://github.com/OpenMRBenelux/openmrb2020-hackathon/issues/7) (please see this link for further details). In this event, I am planning to work on adding multi-phase explosions.

**Good first issues**:
1. Install [Slowest particle simulator on earth](https://github.com/ofgulban/slowest-particle-simulator-on-earth).
2. Use it to explode a brain image that you want to explode. Some options might be:
    - A bad subject who moved too much.
    - Your supervisor's brain.
    - Maybe even your own brain.

**Skills**:
Python, enjoying visual effects, and optionally C, C++, or Cython for understanding the source material mentioned in [here](https://github.com/OpenMRBenelux/openmrb2020-hackathon/issues/7) (please see this link for further details).

**Chat channel**:
Mattermost channel `brainexplosions`

**Video channel**:
TODO: Add when prepared.

## Submission checklist
*Once the issue is submitted, please check items in this list as you add under 'Additional project info'*

Please include the following above (all required):
-   [x] Link to your project: could be a code repository, a shared document, etc. See [here](https://github.com/ohbm/hackathon2020/blob/master/.github/ISSUE_TEMPLATE/handbooks/projects.md#link-to-project)
-   [x] Include your Mattermost handle]. If you do not have an account, please [sign up here](https://mattermost.brainhack.org/signup_email).
-   [x] Goals for the OHBM Brainhack: describe what you want to achieve during this brainhack. See [here](https://github.com/ohbm/hackathon2020/blob/master/.github/ISSUE_TEMPLATE/handbooks/projects.md#goals).
-   [x] Flesh out at least 2 "good first issues": those are tasks that do not require any prior knowledge about your project, could be defined as issues in a GitHub repository, or in a shared document, cf [here](https://github.com/ohbm/hackathon2020/blob/master/.github/ISSUE_TEMPLATE/handbooks/projects.md#onboarding-2-good-first-issues).
-   [x] Skills: list skills that would be particularly suitable for your project. We ask you to include at least one non-coding skill, cf. [here](https://github.com/ohbm/hackathon2020/blob/master/.github/ISSUE_TEMPLATE/handbooks/projects.md#onboarding-skills).
-   [x] Chat channel: A link to a chat channel that will be used during the OHBM Brainhack. This can be an existing channel or a new one. We recommend using the [Brainhack space on mattermost](https://mattermost.brainhack.org/), cf. [here](https://github.com/ohbm/hackathon2020/blob/master/.github/ISSUE_TEMPLATE/handbooks/projects.md#chat).
-   [ ] Video channel: Please create a video channel that will be used during the OHBM Brainhack and share it in your chat channel above. This can be an existing channel or a new one. For instance a [jitsi meet](https://meet.jit.si/) room, cf. [here](https://github.com/ohbm/hackathon2020/blob/master/.github/ISSUE_TEMPLATE/handbooks/projects.md#video-calls).

You can also include information about (all optional):
-   [ ] Number of participants, cf. [here](https://github.com/ohbm/hackathon2020/blob/master/.github/ISSUE_TEMPLATE/handbooks/projects.md#participant-capacity)
-   [ ] Twitter-size summary of your project pitch, cf. [here](https://github.com/ohbm/hackathon2020/blob/master/.github/ISSUE_TEMPLATE/handbooks/projects.md#twitter-size-summary-of-your-project-pitch)
-   [ ] Provide an image of your project for the OHBM brainhack website
-   [ ] Set up a kanban board on your repository to better divide the work and keep track of things, cf [here](https://github.com/ohbm/hackathon2020/blob/master/.github/ISSUE_TEMPLATE/handbooks/projects.md#set-up-a-kanban-board)
-   [ ] Project snippet for the OHBM Brainhack website, cf. [here](https://github.com/ohbm/hackathon2020/blob/master/.github/ISSUE_TEMPLATE/handbooks/projects.md#project-snippet-for-the-ohbm-brainhack-website)

We would like to think about how you will credit and onboard new members to your project. We recommend reading references from [this section](https://github.com/ohbm/hackathon2020/blob/master/.github/ISSUE_TEMPLATE/handbooks/projects.md#credit-and-onboarding). If you'd like to share your thoughts with future project participants, you can include information about (recommended):
-   [ ] Specify how will you acknowledge contributions (e.g. listing members on a contributing page).
-   [ ] Provide links to onboarding documents if you have some.

**Comment by Remi-Gau** [comment]
hey @ofgulban 

Glad to see that you are furthering your pioneering work in brainexplosology.

This looks good to me.

Added some labels on this. Double check that you are ok with those and if there are not others that should be added in our long list

**Comment by Remi-Gau** [comment]
@DorienHuijser I think this can definitely go on the website. :smile:

**Comment by DorienHuijser** [comment]
@Remi-Gau project does not appear on the [board](https://github.com/ohbm/hackathon2020/projects/4) but has been added (see pull request #125)

**Comment by Remi-Gau** [comment]
Thanks @DorienHuijser
Doing github stuff before my second coffee is the best way to make sure i forget about my own workflow.

**Comment by ofgulban** [comment]
Thanks @Remi-Gau , the labels look good to me :).
And thanks @DorienHuijser for taking care of the board issue.

**Comment by Remi-Gau** [comment]
@ofgulban 

Given your schedule, would you prefer to pitch your project on the Americas hub schedule?

**Comment by ofgulban** [comment]
@Remi-Gau , yes pitching on the Americas hub schedule would be more convenient for me. Thanks.

**Comment by Remi-Gau** [comment]
> @Remi-Gau , yes pitching on the Americas hub schedule would be more convenient for me. Thanks.

OK will inform the Americas hub team.

**Comment by ufangYang** [comment]
hi @ofgulban,  Im Yu-Fang who is taking care of your project for the @ohbm/hack2020-social_media :)

**Comment by ofgulban** [comment]
Hi @ufangYang , thanks a lot for the information :) .
Let me know if I need to do something about this.

**Comment by ufangYang** [comment]
> Hi @ufangYang , thanks a lot for the information :) .
> Let me know if I need to do something about this.
Welcome @ofgulban!  Check out your inbox in mattermost :)

**Comment by jhlegarreta** [comment]
@ofgulban were you able to set up a video channel for your project? Thanks !

**Comment by ofgulban** [comment]
Hi @jhlegarreta , I did not set it up yet. I am a bit unsure about which service to use. Any suggestions?
I am closer to Jitsi but had a few bad connectivity experiences with it recently.

**Comment by jhlegarreta** [comment]
> Hi @jhlegarreta , I did not set it up yet. I am a bit unsure about which service to use. Any suggestions?
I am closer to Jitsi but had a few bad connectivity experiences with it recently.

I guess no service is bullet-proof, especially in a free plan. Most projects seem to have chosen jitsi, though. Beyond the more known google meet, skype or zoom, I've seen some hacktrack project suggesting to use discord. To be honest, I find it convenient when there is no need to install any third-party software or there is no need to sign up.

In any case, post the link **only** in the mattermost channel, and pin the message so that contributors can easily find it :+1: !

**Comment by ofgulban** [comment]
Thanks, I will post the link to the chat channel, and pin it. Jitsi is advantageous in a way that it does not require any installation. Maybe I will try that first. Thanks for the suggestions.

**Comment by complexbrains** [comment]
Dear @ofgulban Would you mind giving us a shout when you created and shared the video chat room link and in your Mattermost channel please. Then we will complete the revision and your project will be ready to go! Thank you :hugs:

**Comment by ofgulban** [comment]
Hi @complexbrains , I have posted the video channel on my mattermost channel now. Thanks.

**Comment by complexbrains** [comment]
Thank you @ofgulban! Everything looks ready to go! Then welcome aboard! :hugs: 🎊
